lib/puppet/provider/jail/pyiocage.rb

```diff
@@ -44,30 +59,38 @@ def self.prefetch(resources)
   end
 
   def self.instances
+    default_properties = get_jail_properties('default')
```

Comment:
Just to be clear, I shouldn't actually need a jail named default, right?  Its just a keyword to list the default settings for a would-be-jail.